### PR TITLE
Add API to disable geometric transformations in RandAugment

### PR DESCRIPTION
**Pull request by LukeWood**

### CODE REVIEW

**Comment by LukeWood** [comment]
@qlzh727 @fchollet we can use this in the OD tutorial to boost performance once merged

File: keras_cv/layers/preprocessing/rand_augment.py [hunk 1]
```diff
@@ -63,7 +63,8 @@ class RandAugment(RandomAugmentationPipeline):
             the rate to 10/11 in our implementation, the behavior is identical to
             sampling an Identity augmentation 10/11th of the time.
             Defaults to `1.0`.
-
+        geometric: whether or not to include geometric augmentations.  This should be
```
Comment (qlzh727) [inline keras_cv/layers/preprocessing/rand_augment.py:66]: Please state the type and default value in the docstring.

File: keras_cv/layers/preprocessing/rand_augment.py [hunk 2]
```diff
@@ -63,7 +63,8 @@ class RandAugment(RandomAugmentationPipeline):
...
+            set to False when performing object detection.
```
Comment (qlzh727) [inline keras_cv/layers/preprocessing/rand_augment.py]: Also what if user forget to change this to False when they have bbox? Are we raising any meaningful error?

Comment (LukeWood) [reply keras_cv/layers/preprocessing/rand_augment.py]: They'll get an error that Augment_bounding_box is not implemented - once it is implemented for Shear and translate setting this to true will be valid.

**Comment by LukeWood** [comment]
Thanks @qlzh727, comments addressed

**Review by qlzh727 — APPROVED** [review]

**Pull request merged**